modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/rel/IgniteTableModify.java

```diff
@@ -117,6 +117,8 @@ public IgniteRel clone(RelOptCluster cluster, List<IgniteRel> inputs) {
     public RelWriter explainTerms(RelWriter pw) {
         return super.explainTerms(pw)
                 .itemIf("tableId", getTable().unwrap(InternalIgniteTable.class).id().toString(),
+                        pw.getDetailLevel() == ALL_ATTRIBUTES)
```

Comment:
Why `ProjectableFilterableTableScan#explainTerms` uses condition `pw.getDetailLevel() != EXPPLAN_ATTRIBUTES` but here `pw.getDetailLevel() == ALL_ATTRIBUTES`?

Reply:
It's kinda mess here. In fact, we are using `ALL_ATTRIBUTES` in plan serialisation as well as in plan explanation. So, I'he changed `ProjectableFilterableTableScan#explainTerms` to be consistent with TableModify rel.